dockerfiles/deb.dockerfile

```diff
@@ -125,11 +125,11 @@ RUN ctr --version
 RUN runc --version
 
 FROM scratch AS packages
-COPY --from=build-packages  /archive /archive
-COPY --from=verify-packages /build   /build
+COPY --link --from=build-packages  /archive /archive
+COPY --link --from=verify-packages /build   /build
 
 # This stage is mainly for debugging (running the build interactively with mounted source)
 FROM build-env AS runtime
 ENV GOTOOLCHAIN=local
-COPY --from=golang /usr/local/go/ /usr/local/go/
-COPY common/containerd.service common/containerd.toml /root/common/
+COPY --link --from=golang /usr/local/go/ /usr/local/go/
+COPY --link common/containerd.service common/containerd.toml /root/common/
```

Comment:
I _think_ these should both work fine;

- `/usr/local/go/` shouldn't be present in the base image
- `/root/` directory may be present, but as we're copying _files_ (not directories), I _think_ `--link` will just merge those into what's there (or does `--link` create a layer with an `opaque` to hide underlying directory content; do you know @crazy-max ?)